xion/assetlist.json

```diff
@@ -3,45 +3,368 @@
   "chain_name": "xion",
   "assets": [
     {
-      "description": "XION is the first walletless L1 blockchain purpose built for consumer adoption through chain abstraction.",
-      "extended_description": "XION is the first walletless L1 blockchain purpose built for consumer adoption through chain abstraction. Utilizing protocol-level implementations related to abstracted accounts, signatures, fees, interoperability, and more, XION empowers developers to build secure, intuitive, and seamless user experiences.",
+      "description": "Osmosis from Osmosis Chain",
       "denom_units": [
         {
-          "denom": "uxion",
+          "denom": "ibc/0471F1C4E7AFD3F07702BEF6DC365268D64570F7C1FDC98EA6098DD6DE59817B",
           "exponent": 0,
-          "aliases": ["microxion"]
+          "aliases": [
+            "uosmo"
+          ]
         },
         {
-          "denom": "XION",
-          "exponent": 6,
-          "aliases": ["xion"]
+          "denom": "osmo",
+          "exponent": 6
         }
       ],
-      "base": "uxion",
-      "display": "XION",
-      "name": "xion",
-      "symbol": "XION",
```

Comment:
Since XION is the native staking and fee token, please keep it at top of the list of Assets.